modules/sdk/project-templates-npm-billboardjs-portlet/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,40 @@
+#parse( "definitions.vm" )
+
+<%@ include file="/init.jsp" %>
+
+<!-- Temporary workaround to obtain the library stylesheets -->
+<link href="/o/${artifactId}/node_modules/billboard.js@1.1.1/dist/billboard.css" rel="stylesheet">
+
+<aui:script require="${artifactId}@${packageJsonVersion}">
```

Comment:
@izaera: Don't we usually put `<aui:script>` at the end of a JSP?

Reply:
Again, that's something that depends on what you want to achieve, but we can move it to the end too. In this case it could even make more sense than having it at the top.

Reply:
Ok, I'll move it to the end 😊